package.json

```diff
@@ -1,8 +1,12 @@
 {
   "name": "root",
   "private": true,
```

Comment:
This is a pattern that tells yarn which folder we are using for workspaces.  When you run `yarn install` in the root it will check the package.json files of each of these workspaces and install the shared modules in the root directory